cypress/e2e/court-cases/CourtCaseDetails.Notes.cy.ts

```diff
@@ -158,6 +182,114 @@ describe("Court case details", () => {
+  it("should not display the notes text area if the case is locked to another users", () => {
+    cy.task("insertCourtCasesWithNotesAndLock", {
+      caseNotes: [[]],
+      force: "02"
+    })
+    cy.login("bichard02@example.com", "password")
+    cy.visit("/bichard/court-cases/0")
+
+    clickTab("Notes")
+    cy.contains("Case has no notes.")
+  })
+
+  it("should display the notes text area if the case is not locked to another users", () => {
+    cy.task("insertCourtCasesWithNotes", {
+      caseNotes: [[]],
+      force: "02"
+    })
+    cy.login("bichard02@example.com", "password")
+    cy.visit("/bichard/court-cases/0")
+
+    clickTab("Notes")
+    cy.get("label").contains("Add a new note")
+    cy.get("textarea").should("be.visible")
+    cy.get("span").contains("You have 1000 characters remaining")
+  })
+
+  it("should be able to add a note when case is visible to the user and not locked by another user", () => {
+    insertTriggers()
+    loginAndGoToNotes()
+    cy.get("textarea").type("Dummy note")
+    cy.get("button").contains("Add note").click()
+
+    clickTab("Notes")
+
+    const dateTimeRegex = /\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}/
+    cy.contains(dateTimeRegex)
+    cy.contains("Dummy note")
+  })
+
+  it("should be able to add a long note", () => {
+    insertTriggers()
+    loginAndGoToNotes()
+    cy.contains("h3", "Notes")
+
+    cy.get("textarea").type("A ".repeat(500), { delay: 0 })
+    cy.get("button").contains("Add note").click()
+
+    cy.get("textarea").type("B ".repeat(500), { delay: 0 })
+    cy.get("button").contains("Add note").click()
+
+    cy.get("textarea").type("C ".repeat(100), { delay: 0 })
+    cy.get("button").contains("Add note").click()
+
+    cy.get("H1").should("have.text", "Case details")
+    clickTab("Notes")
+
+    cy.contains("A ".repeat(500))
+    cy.contains("B ".repeat(500))
+    cy.contains("C ".repeat(100))
+  })
+
+  it("should show error message when note text is empty", () => {
+    insertTriggers()
+    loginAndGoToNotes()
+
+    cy.get("H3").contains("Notes")
+    cy.get("button").contains("Add note").click()
+    cy.get("form span").contains("Input message into text box")
+    cy.get("textarea[name=noteText]").type("dummy note")
+    cy.get("form span").should("not.contain", "Input message into text box")
+    cy.get("button").contains("Add note").click()
+
+    cy.contains("dummy note")
+  })
+
+  it("Adding an empty note doesn't add a note, when the case is visible to the user and not locked by another user", () => {
+    insertTriggers()
+    loginAndGoToNotes()
+
+    cy.url().should("match", /.*\/court-cases\/0/)
+    cy.get("H1").should("have.text", "Case details")
+    clickTab("Notes")
+    cy.findByText("Case has no notes.").should("exist")
+  })
+
+  it("should return 404 for a case that this user can not see", () => {
+    cy.task("insertCourtCasesWithFields", [{ orgForPoliceFilter: "02" }])
+    cy.login("bichard01@example.com", "password")
+
+    cy.request({
```

Comment:
I don't think these tests are relevant anymore since the notes are in a tab on the case page and not a separate page. We already test this on the case details page at https://github.com/ministryofjustice/bichard7-next-ui/blob/8b53cea5825af957ccafe65596bb55befffb05cb/cypress/e2e/court-cases/CourtCaseDetails.cy.ts#L184